modules/update-upgrading-cli.adoc

```diff
@@ -4,6 +4,24 @@
 // * updating/updating-cluster-rhel-compute.adoc
 
 [id="update-upgrading-cli_{context}"]
+= Optional: Changing the update server by using the CLI
+
+If you have an OpenShift Update Service (OSUS) installed and configured locally, you can use it. To consume your local OSUS, you must change the upstream. The default `upstream` is https://api.openshift.com/api/upgrades_info/v1/graph.
+
+* Change the `upstream` parameter value in the cluster version:
++
+[source,terminal]
+----
+$ oc patch clusterversion/version --patch '{"spec":{"upstream":"https://example.com/graph"}}' --type=merge
```

Comment:
Checking our official doc, looks like when we configure something, we usually use a variable. So here, how about using a variable to replace https://example.com/graph. And add a description to let users know to replace the variable with their update server URI.

Reply:
Done. Please let me know if any other change is required.